crates/matrix-sdk/src/room/mod.rs

```diff
@@ -3244,6 +3244,9 @@ impl Room {
     ///
     /// The returned observable will receive the newest event for each sync
     /// response that contains an `m.beacon` event.
+    ///
+    /// Returns a stream of `ObservableLiveLocation` events from other users
```

Comment:
Let's linkify this, makes things much more pleasant to read and you can always quickly go to the docs of the referenced type.

```suggestion
    /// Returns a stream of [`ObservableLiveLocation`] events from other users
```